### PR TITLE
Bug/timeline-attachments

### PR DESCRIPTION
**Pull request by privatmamtora**
If the Attachment is for example a video, it appears as a broken image.
The content type is being checked with a `startsWith("")`, when it is supposed to be `startsWith("image")`

Also made images clickable.

### CODE REVIEW

File: src/main/webapp/index.jsp [hunk 1]
```diff
@@ -109,9 +109,12 @@ limitations under the License.
                 <%
                 if (timelineItem.getAttachments() != null) {
                   for (Attachment attachment : timelineItem.getAttachments()) {
-                    if (MirrorClient.getAttachmentContentType(credential, timelineItem.getId(), attachment.getId()).startsWith("")) { %>
+                    if (MirrorClient.getAttachmentContentType(credential, timelineItem.getId(), attachment.getId()).startsWith("image")) { %>
+                <a href="<%= appBaseUrl + "attachmentproxy?attachment=" +
+                  attachment.getId() + "&timelineItem=" + timelineItem.getId() %>">
                 <img src="<%= appBaseUrl + "attachmentproxy?attachment=" +
```
Comment (alainv) [inline src/main/webapp/index.jsp]: Can you indent this line and the following two properly?
    <a ...>
      
    </a>

**Comment by alainv** [comment]
Thanks for the pull request, please fix the comment and make sure to sign the [CLA](https://github.com/googleglass/mirror-quickstart-java/blob/master/CONTRIBUTING.md) before I can merge.

File: src/main/webapp/index.jsp (hunk 1 above)
Comment (privatmamtora) [inline src/main/webapp/index.jsp]: Done

**Comment by privatmamtora** [comment]
Already signed CLA

**Pull request merged**